src/assets.ts

```diff
@@ -488,7 +481,18 @@ function addLaunchJsonIfNecessary(generator: AssetGenerator, operations: Operati
 
         const isWebProject = generator.hasWebServerDependency();
         const launchJson = generator.createLaunchJson(isWebProject);
-        const launchJsonText = JSON.stringify(launchJson, null, '    ');
+
+        const configurationsMassaged = JSON.stringify(launchJson, null, '   ').split('\n').map(line => '   ' + line).join('\n').trim();
+
+        const launchJsonText = [
+			'{',
+			'\t// Use IntelliSense to find out which attributes exist for C# debugging',
+			'\t// Use hover for the description of the existing attributes',
+			'\t// For further information visit https://go.microsoft.com/fwlink/?linkid=830387',
```

Comment:
I think we should come up with our own URL/Documentation. This is the generic VS Code one which isn't nearly as nice as pointing at something C#-extension specific. I will start a mail thread and include you.